ecep-cms/settings_custom.py

```diff
@@ -79,3 +79,62 @@
 ########################
 
 INCLUDES_CORE_PORTAL = True
+
+########################
+# TACC: NEWS/BLOG
+########################
+
+# TACC settings
+TACC_BLOG_SHOW_CATEGORIES = False
+TACC_BLOG_SHOW_TAGS = False
```

Comment:
**Note.** ECEP does not use categories nor tags, and has no immediate interest in doing so. Other projects could be similar.

Reply:
I like the flexibility!